src/mono/mono/eglib/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+project(eglib)
```

Comment:
nit: we can also add `C` language to the relevant `project` directives. e.g. https://github.com/dotnet/runtime/blob/33e912de1027bc262ecc8ba6642359508fffc703/src/native/libs/System.IO.Compression.Native/CMakeLists.txt#L1 it makes sure that when building this project, only initialize the required toolchain instead of "all default toolchain" (in most cases under `src/mono`, C++ is not needed)

Reply:
ok, I set it.  The interesting one is `mini`. I did `project(mini C)` and then a separate `enable_language(CXX)` under some conditions.